### PR TITLE
Improved Translation Card Positioning for Consistent UX

### PR DESCRIPTION
**Pull request by yizhilee**
Hello Yetone,

I have made some changes to the translation card's positioning in the openai-translator Chrome browser extension. With the help of ChatGPT, I was able to modify the card to consistently appear below the selected text, providing a more predictable and consistent user experience.

I have never developed a Chrome extension before, but with the guidance of ChatGPT, I successfully made these improvements to the tool. It's quite dramatic how I, a complete beginner in Chrome extension development, could enhance this ChatGPT-based tool with the AI's assistance.

Please consider merging these changes to improve the user experience for everyone using the openai-translator extension. I am open to any feedback and suggestions you might have. Thank you for your time and consideration.

On a lighter note, it's interesting to mention that this comment was also written with the help of ChatGPT!

Best regards

### CODE REVIEW

File: src/content_script/index.tsx [hunk 1]
```diff
@@ -22,9 +22,16 @@ async function popupThumbClickHandler(event: MouseEvent) {
     if (!$popupThumb) {
         return
     }
-    const x = $popupThumb.offsetLeft
-    const y = $popupThumb.offsetTop
-    showPopupCard(x, y, $popupThumb.dataset['text'] || '')
+
+    const range = window.getSelection()?.getRangeAt(0)
+    const rect = range?.getBoundingClientRect()
+    const popupWidth = 600
```
Comment (yetone) [inline src/content_script/index.tsx:28]: Where does this magic number come from and can it be used as a global variable?

**Comment by yetone** [comment]
I feel that the position where pop-ups appear is a matter of preference. The word translation software I am used to using displays pop-ups according to the location of the mouse pointer, so can this be made into a configuration option?

File: src/content_script/index.tsx (hunk 1 above)
Comment (yizhilee) [reply src/content_script/index.tsx:28]: This is the width of the card that I obtained from inspecting elements, perhaps we have a better variable to replace it.

**Comment by yizhilee** [comment]
> I feel that the position where pop-ups appear is a matter of preference. The word translation software I am used to using displays pop-ups according to the location of the mouse pointer, so can this be made into a configuration option?

Sounds good, this way users can decide for themselves.

**Pull request closed without merging**